general-superstaq/general_superstaq/check/coverage_.py

```diff
@@ -13,6 +13,7 @@ def run(
     *args: str,
     include: Union[str, Iterable[str]] = "*.py",
     exclude: Union[str, Iterable[str]] = "*_integration_test.py",
+    namespace: argparse.Namespace = argparse.Namespace(),
```

Comment:
is there any reason to pass this around anymore?

if so, this should just be
```suggestion
    namespace: Optional[argparse.Namespace] = None,
```

Reply:
Oh nope, my mistake.

Reply:
removed